akkeeper/src/main/resources/reference.conf

```diff
@@ -121,7 +121,7 @@ akkeeper {
     servers = ""
 
     # The initial amount of time to wait between retries.
-    connection-interval-ms = 3000
+    connection-interval-ms = 500
```

Comment:
Why did you change it?

Reply:
in `ZookeeperClient` we have ` .retryPolicy(new ExponentialBackoffRetry(config.connectionIntervalMs, config.maxRetries))` - so we start from 3000ms and then exponentially backoff up to `max-retries`. 3000 ms * (2 ^ 8) is already around 12 minutes - way over our timeout. 500ms brings that time to around 2 minutes max